lib/node_modules/@stdlib/math/base/special/besselj0/lib/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+* Compute the Bessel function of the first kind of order zero.
+*
+* @module @stdlib/math/base/special/besselj0
+*
+* @example
+* var j0 = require( '@stdlib/math/base/special/besselj0' );
+*
+* var v = j0( 0.0 );
+* // returns 1.0
+*
+* v = j0( 1.0 );
+* // returns ~0.765
+*
+* v = j0( Infinity );
```

Comment:
Since you used `PINF` and `NINF` in the implementation file examples, also require them here.